Source/ACE.Server/Managers/LandblockManager.cs

```diff
@@ -153,7 +153,7 @@ public static Landblock GetLandblock(LandblockId landblockId, bool loadAdjacents
             {
                 var adjacents = GetAdjacentIDs(landblock);
                 foreach (var adjacent in adjacents)
-                    GetLandblock(adjacent, false, false, false);
+                    GetLandblock(adjacent, false);
```

Comment:
This re-introduces the invisible player bug

The landblock loading needs a callback for when CreateWorldObjects() as completed